Digipost.Api.Client.Shared/Certificate/CertificateValidator.cs

```diff
@@ -174,9 +175,17 @@ private static CertificateValidationResult ValidResult(X509Certificate2 certific
                 certificate.ToShortString(CertificateValidResult));
         }
 
-        private static bool IsIssuedToOrganizationNumber(X509Certificate certificate, string certificateOrganizationNumber)
+        private static bool IsIssuedToOrganizationNumber(X509Certificate certificate,
+            string certificateOrganizationNumber)
         {
-            return certificate.Subject.Contains($"SERIALNUMBER={certificateOrganizationNumber}") || certificate.Subject.Contains($"CN={certificateOrganizationNumber}");
+            // SEID 2 way to embed Norwegian "organisasjonsnummer" in certificates.
+            //  
+            // @see <a href="http://www.oid-info.com/get/2.5.4.97">OID 2.5.4.97</a>
+            string seid2OrgNumberPattern = "(OID\\.2\\.5\\.4\\.97|organizationIdentifier)=(?:NTRNO-)?([0-9]{9})";
+            
+            return certificate.Subject.Contains($"SERIALNUMBER={certificateOrganizationNumber}")
+                   || certificate.Subject.Contains($"CN={certificateOrganizationNumber}")
+                   || Regex.IsMatch(certificate.Subject, seid2OrgNumberPattern);
```

Comment:
Vil ikke dette bare sjekke at `certificate.Subject` matcher _patternet_ med et eller annet organisasjonsnummer, men ikke nødvendigvis for gitt `certificateOrganizationNumber`?

Reply:
Haha, det stemmer ja 🤦  Fikser det her f7a4937